docs/dapps/oev-rewards/dapp-id.md

```diff
@@ -0,0 +1,50 @@
+---
+title: dApp ID
+pageHeader: dApps → OEV Rewards
+---
+
+<PageHeader/>
+
+# dApp ID
+
+API3 uses a dApp ID to identify a dApp on a specific chain, similar to how chain IDs are used to identify networks.
+It is a 256-bit unsigned integer, and is assigned to dApps by API3 in an incremented way with no specific order.
+The dApp ID of a dApp is static, it never changes.
+
+## dApp ID list
+
+| dApp ID | dApp alias            | Network            |
+| ------- | --------------------- | ------------------ |
+| 1       | API3 Market (default) | All networks       |
+| 2       | MyPlaceholderDapp     | Ethereum (1)       |
+| 3       | MyPlaceholderDapp     | Gnosis Chain (100) |
```

Comment:
TODO: Update these before merging

Reply:
@Ashar2shahid get me an up to date list of current dApps, on which chains they use us, what data feeds they use on each